### PR TITLE
Customer show page

### PR DESCRIPTION
**Pull request by gotoinc**

### CODE REVIEW

File: app/controllers/customers_controller.rb [hunk 1]
```diff
@@ -61,13 +62,99 @@ def update
   # DELETE /customers/1 or /customers/1.json
   def destroy
     CompanyCustomer.find_by(company_id: current_user.company_id, customer_id: @customer.id).destroy
+    redirect_to customers_path, notice: "Customer was successfully removed."
+  end
+
+  def new_address
```
Comment (iferunsewe) [inline app/controllers/customers_controller.rb:68]: This should be in it's own "AddressController"?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 module ApplicationHelper
+
+  def prepend_flash
```
Comment (iferunsewe) [inline app/helpers/application_helper.rb:3]: Why is this needed?

**Review by iferunsewe — CHANGES_REQUESTED** [review]
The address and customer server code should be seperated

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (gotoinc) [reply app/helpers/application_helper.rb:3]: This helper allows me to show flash with turbo_stream response. I see that I wrote this helper only for div#flash2. I can add tag_id variable to do this method more reusable. Also, suggest renaming flash2 to addresses_flash.

Comment (iferunsewe) [reply app/helpers/application_helper.rb:3]: Lets rename it to `prepend_flash_to_turbo` as it won't just be used for `addresses`

File: app/controllers/customers/addresses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,98 @@
+module  Customers
+  class AddressesController < ::ApplicationController
+    before_action :set_customer
+    def new_address
```
Comment (iferunsewe) [inline app/controllers/customers/addresses_controller.rb:4]: Can these just be the normal action names like e.g. "new", "create", "edit" , "update", "destroy"

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (gotoinc) [reply app/helpers/application_helper.rb]: okay

**Review by iferunsewe — APPROVED** [review]

**Pull request merged**